README.md

```diff
@@ -86,12 +86,12 @@ FLAGS:
     -V, --version    Prints version information
 
 OPTIONS:
-        --subgraph <IPFS_HASH>       IPFS hash of the subgraph manifest
-        --ethereum-ipc <FILE>        Ethereum IPC pipe
-        --ethereum-rpc <URL>         Ethereum RPC endpoint
-        --ethereum-ws <URL>          Ethereum WebSocket endpoint
-        --ipfs <HOST:PORT>           HTTP address of an IPFS node
-        --postgres-url <URL>         Location of the Postgres database used for storing entities
+        --subgraph <[NAME:]IPFS_HASH>         Name (optional) and IPFS hash of the subgraph manifest
+        --ethereum-ipc <NETWORK_NAME:FILE>    Ethereum network name (e.g. 'mainnet') and Ethereum IPC pipe path, separated by a ':'
+        --ethereum-rpc <NETWORK_NAME:URL>     Ethereum network name (e.g. 'mainnet') and Ethereum RPC endpoint URL, separated by a ':'
+        --ethereum-ws <NETWORK_NAME:URL>      Ethereum network name (e.g. 'mainnet') and Ethereum WebSocket endpoint URL, separated by a ':'
```

Comment:
Would these be clearer as `<NETWORK_NAME>:<URL>`?

Reply:
Yeah, could be. But then how do we represent the optional params? `<[NAME:]IPFS_HASH>`

Reply:
Hmm true. `[<NAME>]:<IPFS_HASH>`?
or could take off `< >` altogether, maybe that's better